ldp/alg/algorithms.py

```diff
@@ -190,3 +191,25 @@ async def evaluate_consensus(
             ideal_count += consensus == ideal_answer_fn(group[0])
 
     return grouped_consensus, ideal_count / len(groups) if groups else 0.0
+
+
+def compute_pass_at_k(n: int, c: int, k: int) -> float:
+    """Compute an unbiased estimation for 'pass @ k'.
+
+    Source: https://doi.org/10.48550/arXiv.2107.03374
```

Comment:
Would it be worth mentioning Figure 3 of the paper here?

Reply:
Sure, I did this in https://github.com/Future-House/ldp/pull/203